Gemfile

```diff
@@ -41,5 +41,15 @@ group :test do
   gem 'webdrivers'
```

Comment:
For what this dependecy?

Reply:
This dependency was not removed by cleaning before intitial commit. I'll create another PR with deleting all not needed stuff